README.md

```diff
@@ -11,7 +11,7 @@ Draw SVG files using Flutter.
 ## Getting Started
 
 This package provides a wrapper around Dart implementations of SVG parsing,
-including SVG path data. In particular, it provides efficient `BytesLoader`
+including SVG path data. In particular, it provides an efficient `BytesLoader`
```

Comment:
This isn't right. Implementations is plural.

Reply:
Ah, I see, removed the added "an".